package.json

```diff
@@ -1,15 +1,44 @@
 {
-  "name": "holdings",
-  "version": "0.1.0",
+  "name": "photo-gallery",
```

Comment:
need to change this to appropriate project name  . @sachinrath1986  how about wissenPortfolioApp ?